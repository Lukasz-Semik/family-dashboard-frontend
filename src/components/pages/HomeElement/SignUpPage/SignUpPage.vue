<template>
  <UserSign title-translation-path="general.signUp">
    <div :class="[$style['form-wrapper']]">
      <SignUpForm/>
    </div>
  </UserSign>
</template>

<script>
import UserSign from '@/components/molecules/UserSign/UserSign.vue';
import SignUpForm from '@/components/organisms/SignUpForm/SignUpForm.vue';

export default {
  components: {
    UserSign,
    SignUpForm,
  },
};
</script>

<style lang="scss" module src="./SignUpPage.scss" />
