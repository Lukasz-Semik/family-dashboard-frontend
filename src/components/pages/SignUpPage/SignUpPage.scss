.form-wrapper {
  margin: 0 auto;
  width: 40rem;
  max-width: 90%;
}
