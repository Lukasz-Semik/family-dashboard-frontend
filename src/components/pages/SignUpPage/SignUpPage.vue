<template>
  <Title translationPath="general.testSignUpPage"></Title>
</template>

<script>
import Title from '@/components/atoms/Title/Title.vue';

export default {
  components: {
    Title,
  },
};
</script>
