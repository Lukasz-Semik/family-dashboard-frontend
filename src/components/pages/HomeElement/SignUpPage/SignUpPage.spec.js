import { shallowMount } from '@vue/test-utils';

import SignUpPage from './SignUpPage.vue';

describe('<SignUpPage/>', () => {
  it('should render properly', () => {
    const wrapper = shallowMount(SignUpPage);

    expect(wrapper).toMatchSnapshot();
  });
});
