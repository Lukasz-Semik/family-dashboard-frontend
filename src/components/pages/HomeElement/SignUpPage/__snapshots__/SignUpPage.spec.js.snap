// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SignUpPage/> should render properly 1`] = `
<usersign-stub titletranslationpath="general.signUp">
  <div class="form-wrapper">
    <signupform-stub></signupform-stub>
  </div>
</usersign-stub>
`;
